webots_ros2_epuck/README.md

```diff
@@ -205,4 +205,29 @@ t1$ ros2 run nav2_map_server map_server --ros-args -p yaml_filename:=$HOME/Pictu
 t2$ ros2 service call /map_server/change_state lifecycle_msgs/ChangeState "{transition: {id: 1}}"
 t2$ ros2 service call /map_server/change_state lifecycle_msgs/ChangeState "{transition: {id: 3}}"
 t2$ ros2 launch webots_ros2_epuck example_launch.py rviz:=true nav:=true
-```
\ No newline at end of file
+```
+
+### Differential Drive Calibration
+Based on the rotation speed of each wheel and two constants, distance between the wheels and wheel radius, we can calculate the position of the robot in the local frame. Therefore. the precision of the position estimation depends a lot on the distance between the wheels and wheel radius. Those constants can vary from robot to robot and here we provide a tool to help you to calibrate it (this technique is very similar to one proposed in [_"Measurement and Correction of Systematic Odometry Errors in Mobile Robots"_](http://www-personal.umich.edu/~johannb/Papers/paper58.pdf)).  
```

Comment:
```suggestion
Based on the rotation speed of each wheel and two constants, distance between the wheels and wheel radius, we can calculate the position of the robot in the local frame.
Therefore, the precision of the position estimation depends a lot on the distance between the wheels and the wheel radius.
Those constants can vary from robot to robot and here we provide a tool to help you to calibrate it (this technique is very similar to the one proposed in [_"Measurement and Correction of Systematic Odometry Errors in Mobile Robots"_](http://www-personal.umich.edu/~johannb/Papers/paper58.pdf)).
```

We have a simple rule for avoiding long lines in md files (which can be cumbersome for the review), at each new sentence we add a new line.

Reply:
I have just applied the rule to the rest of the README

Reply:
Perfect, thank you!